### DISCUSSION TITLE
LED sequence of anchor when loco positioning is working well

### DISCUSSION THREAD

**Discussion by AlEXANDER9901** [bitcraze/discussions#564] [Q&A]
Could you tell me how the LED lights up when loco positiong is working fine?
We turn on like the next video, but it doesn't seem to be working properly. I would really appreciate it if you could look at the sequence of lights.
The sync LED does not come on.

thanks a lot!

https://user-images.githubusercontent.com/125459416/219265287-69a1da60-9dd0-48aa-b589-0e1e2e40300c.mp4

**Comment by krichardsson** [comment]
Hi!

The row of LEDs that is blinking in your video roughly means that the radio is transmitting and receiving data. In general terms your anchor looks OK, but the blinking depends on which mode you are running the system in. It all boils down to if there is communication going on or not.

### TWR

In TWR the Crazyflie is initiating all communication, it pings one anchor at a time. This means that if the Crazyflie is turned off, the anchors should not communicate (blink) they are just in receive mode waiting for a packet from the Crazyflie. When you turn on the Crazyflie it will start to range with the anchors and they should start to blink.

### TDoA2

It is better to use TDoA3, I'l skip this

### TDoA3

The anchors are continuously both transmitting and receiving and should blink all the time

### The sync LED

I'm not sure, @ataffanel do you have any input on this?

**Reply by AlEXANDER9901** [reply]
thank you:) That was helpful!

**Comment by ataffanel** [comment]
According to [the specification](https://www.bitcraze.io/products/loco-positioning-node/), the sync LED is not used yet in our firmware. I agree with Kristoffer, this LED pattern looks like the anchor is working well. Maybe you can describe the problem you have and the procedure you have followed to setup the system.

**Reply by AlEXANDER9901** [reply]
thanks for reply.

In fact, we have installed 8 anchors in the room, and we want to fly the drone with Loco positioning, but as shown in the picture, the anchors appear green, but the location is not accurate.
So I wondered if there was a problem with transmitting and receiving data in the anchor.
but thanks to that, it was solved.

Again, we look at the console window to see if there is a problem with the LPS deck. I'd like to find out if there's a procedure we're missing, or if a deck setup is needed.
We think there is a problem with the loco positioning deck mounting, but we are curious because we are not sure.
![사본 -026](https://user-images.githubusercontent.com/125459416/219430428-4c5fad55-0c72-4d04-ba7f-6c0f3d3d20e0.png)
![dhkdhfkdhfkdhfdkhfd](https://user-images.githubusercontent.com/125459416/219430460-cbddfaf2-e74a-40a1-9bde-d0ec28db08b8.png)

As an additional note, in the procedure we followed, there was no special setting for the positioning deck.
https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/howto/
Do I need this course?

I would like it if you could help me.

**Reply by krichardsson** [reply]
Please continue this discussion in the thread (#562) you already started on this topic. (Discussion Rules #55)
